src/index.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
```

Comment:
For all the react imports, you can consider removing them. In the latest version, it's not needed. The reason for this starting from release 17 of React, JSX is automatically transformed without using `React.createElement.`

Reply:
As long as I am using the react strict mode, I think I need to import the react in the index 'cause if I do not import it will result into an error.